config/manifests/bases/network-observability-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: network-observability-operator.v0.0.0
```

Comment:
should we start using versions ? 
at least v0.0.1 ?

Reply:
I agree but since this would require having also versions in the artifact generation and modifying Prow, I have created a task to put all the versioning tasks together: https://issues.redhat.com/browse/NETOBSERV-107

Reply:
ok, good to know :+1: